.github/workflows/sign.yaml

```diff
@@ -3,58 +3,52 @@ name: Test
 # only trigger on pull request closed events
 on:
   pull_request:
-    types: [ closed ]
+    branches:
+      - main
 
 jobs:
   merge_job:
-    # this job will only run if the PR has been merged
-    if: github.event.pull_request.merged == true
     runs-on: ubuntu-latest
     permissions:
       contents: write
       actions: read
       id-token: write # Needed for OIDC and keyless signing
     steps:
-    - run: |
-        echo PR #${{ github.event.number }} has been merged, beginning signing flow
```

Comment:
Why remove this? The signing should only be done when the PR has been approved and merged.